packages/contracts/contracts/TaskRegistry.sol

```diff
@@ -35,9 +35,12 @@ contract TaskRegistry is Ownable, IProposalStruct {
         require(msg.sender == tasks[taskId].issuer, "Not the issuer of the task");
         _;
     }
-    constructor(AgentsRegistry _agentRegistry) Ownable(msg.sender) {
+    constructor(
+        uint256 _nextTaskId, 
```

Comment:
can you please rename it to _initialTaskId

Reply:
nvm let's keep like this for name so we won't have any changes (though this shouldn't change the verificaton)